docs/hardware.md

```diff
@@ -34,6 +34,13 @@ These boards run MakeCode Arcade games. Choose a board to find out more about it
         "url": "https://www.adafruit.com/product/4242",
         "variant": "hw---samd51"
     },
+    {
+        "name": "Kitronik ARCADE",
+        "description": "ARCADE is a programmable gamepad for use with MakeCode Arcade.",
```

Comment:
Different description? This one doesn't sound right.

Reply:
I have asked them for one. Will update when they answer.